archetypes/Exchange/Inputs/index.tsx

```diff
@@ -104,12 +131,12 @@ export default (({ pool, swapState, swapDispatch }) => {
                         swapDispatch({ type: 'setSide', value: parseInt(side) as SideEnum });
                     }}
                 />
-                <p className={classNames(!!pool.address ? 'block' : 'hidden', 'text-sm opacity-70 mt-2')}>
+                <p className={classNames(!!pool.address ? 'block' : 'hidden', 'subtext')}>
```

Comment:
we should try and avoid the tailwind styled-components marriage, where you can just use styled components, Could create a styled.p that has a hidden prop